### ISSUE TITLE
Small Spinner on explorer tree nav bar and document tabs has opacity 0

### ISSUE THREAD

**Issue by at055612** [gchq/stroom#5299]
The spinning icon is constantly spinning on every tab and the exp tree nav bar, but not visible to the user due to the opacity.  Assume it is using CPU to do this when not required, and confusing screen readers.

Suggest changing the spinner.css from
```css
.SpinnerSmall {
    opacity: 0;
}

.SpinnerSmall.refreshing {
    opacity: 1;
}
```

to
```css
.SpinnerSmall {
    visibility: hidden;
}

.SpinnerSmall.refreshing {
    visibility: visible;
}
```
